pkg/proxy/interface.go

```diff
@@ -0,0 +1,17 @@
+package proxy
```

Comment:
Let's make this PR a draft and only have it include this file. It's fine if the Relay type is not defined yet (i.e. it won't compile)